src/main/java/uk/ac/ebi/eva/commons/models/data/Variant.java

```diff
@@ -406,4 +399,32 @@ private String composeId(String studyId, String fileId) {
         return studyId + "_" + fileId;
     }
 
+    public String buildVariantId(){
+        return buildVariantId(chromosome, start, reference, alternate);
+    }
+
+    public static String buildVariantId(String chromosome, int start, String reference, String alternate) {
```

Comment:
Even though I kind of like the instance method that gets rid of the arguments, I think this is too database dependant and it should be moved back to `MongoDBHelper`. Any comments against @jorizci ?